redfish/provider/common.go

```diff
@@ -290,7 +315,7 @@ func (p powerOperator) PowerOperation(resetType string, maximumWaitTime int64, c
 }
 
 // Check checks iDRAC server status after provided interval until the provided timeout time
-func (s *ServerStatusChecker) Check(ctx context.Context) error {
+func (s *ServerStatusChecker) Check(ctx context.Context, sysid string) error {
```

Comment:
This particular function does not care about `sysid` at all... it only checks that the API is working...
So we shd simply use `getSystemResource(s.Service, "")` inside this function.